features/salt_states_catalog.feature

```diff
@@ -42,4 +42,4 @@ Feature: Check the Salt package state UI
     Then I should see a "1 Changes" text
     And I click on the css "button#save-btn"
     And I click on the css "button#apply-btn"
-    Then I wait for the file "/root/foobar"
+    Then "/root/foobar" exists on the filesystem of "sle-minion"
```

Comment:
"I wait for the file" was another function that did something else, why do you removed and replace the function?